known_issues/known_issue_data.csv

```diff
@@ -0,0 +1,6 @@
+# File of known issues in ICU/conformance matrix
```

Comment:
Nit: Don't add files that we aren't using because it is confusing; people will find this file and think it does something but it doesn't

Reply:
Removed for now.